terraso_backend/apps/shared_data/forms.py

```diff
@@ -30,6 +32,7 @@
 logger = structlog.get_logger(__name__)
 
 VALID_CSV_TYPES = ["text/plain", "text/csv", "application/csv"]
+VALID_GEOJSON_TYPES = ["text/plain", "application/json", "application/geo+json"]
```

Comment:
@paulschreiber These constants are used specifically for plain text files since their mime types can vary, this is not the same as the list of accpeted extensions in the settings.py file

Reply:
Understood. Please move the constants to the config file.